news/2021-07-20-project-loved-july-2021.md

```diff
@@ -0,0 +1,267 @@
+160 BPM, 5:02 | 7.90★\
+*description written by [Spectator](https://osu.ppy.sh/users/702598)*
+
+AutoLs is well-known for their crazy playing skill, but people often overlook the fact that they're skillful at mapping as well. This month we have their map of *Hypnotize* going for Loved, which consists of various technical patterns, such as hypersliders and 1/8 bursts. What stands out from the map is the use of consecutive 1/8 hyperdashes in the map, which makes it more interesting to play as the BPM of 160 might be a bit too slow for some top-notch players. Please check out the map if you're looking for further challenge!
+
+[![](/wiki/shared/news/2021-07-20-project-loved-july-2021/166993.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Yueporu - Kurikaeshi Hitotsubu](https://osu.ppy.sh/beatmapsets/166993#fruits) by [Magiyu](https://osu.ppy.sh/users/1667470)\
+236 BPM, 4:08 | 4.28★, 5.21★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+This month we have *Kurikaeshi Hitosubu* going for Loved, made by Magiyu. A mapper you might not know as they are a fairly old mapper (and player!)
+
+The map has two difficulties, both of them shouldn't be *too* hard for most players as they are of a fairly low star rating. Both of the difficulties focus mostly on stair-like patterns which might give you some issues playing through, however are very rewarding to play. Give the map a spin and see what you can achieve!
+
+[![](/wiki/shared/news/2021-07-20-project-loved-july-2021/20296.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[ON-kei - Taihen na Mono no Shoushitsu -GOSSUN END-](https://osu.ppy.sh/beatmapsets/20296#fruits) by [ZHSteven](https://osu.ppy.sh/users/142413), [Kawachino Yumi](https://osu.ppy.sh/users/38805), [pieguyn](https://osu.ppy.sh/users/107485) and [Shino](https://osu.ppy.sh/users/323677)\
+240 – 300 BPM, 2:56 | 3.63★, 3.64★, 5.40★, 5.45★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+Here we have *Taihen na Mono no Shoushitsu*, which is a mapset hosted by ZHSteven, who, albeit not being active anymore, has contributed a lot to osu!catch mapping back in the day. And oh boy, if you thought some of the maps that have been Loved before were old, then this one is ancient. This map was created and last updated in September 2010 (!). However, that does not mean that the difficulties are inherently bad.
+
+For old standards this was pretty good quality compared to most of the other things from around that time. While this mapping style obviously didn't age very well, it is still really fun to play these maps nowadays, as they are challenging in their own way. 'catch the yoki' and 'Steven's Jumping' are not that easy to play and should be tricky for most players that are not used to these types of styles. 'pieguy's CTB' and 'Crow'z Lost' on the other hand, are rather easy to play and a good entry point for people who are not that familiar with old mapping.
```

Comment:
not sure if correct but will lean on consistency

```suggestion
For old standards this was pretty good quality compared to most of the other things from around that time. While this mapping style obviously didn't age very well, it is still really fun to play these maps nowadays, as they are challenging in their own way. "catch the yoki" and "Steven's Jumping" are not that easy to play and should be tricky for most players that are not used to these types of styles. "pieguy's CTB" and "Crow'z Lost" on the other hand, are rather easy to play and a good entry point for people who are not that familiar with old mapping.
```

Reply:
i completely missed this while updating descriptions, i'll fix this on the next commit once forum posts are gonna be ready to go